pkg/crypto/tinkcrypto/primitive/composite/ecdhes/subtle/ecdhes_aes_aead_composite_encrypt.go

```diff
@@ -16,9 +16,13 @@ import (
 	commonpb "github.com/hyperledger/aries-framework-go/pkg/crypto/tinkcrypto/primitive/proto/common_composite_go_proto"
 )
 
-// A256GCM is the default content encryption algorithm value as per
-// the JWA specification: https://tools.ietf.org/html/rfc7518#section-5.1
-const A256GCM = "A256GCM"
+const (
+	// A256GCM is the default content encryption algorithm value as per
+	// the JWA specification: https://tools.ietf.org/html/rfc7518#section-5.1
+	A256GCM = "A256GCM"
+	// DIDComEncType representing the JWE 'Typ' protected type header
+	DIDComEncType = "didcomm-envelope-enc"
```

Comment:
why is this constant duplicated?

Reply:
moved to common package and removed duplicates